.github/CODEOWNERS

```diff
@@ -730,31 +720,30 @@
 #
 # On modifying this list make sure to keep the CODEOWNER_IMPORTANT_PATHS list
 # in scripts/linters/codeowner_linter.py in sync with the modifications.
-/core/templates/services/svg-sanitizer.service.ts @seanlip
-/scripts/linters/warranted_angular_security_bypasses.py @seanlip
-/core/controllers/access_validators*.py @seanlip
-/core/controllers/acl_decorators*.py @seanlip
-/core/controllers/base*.py @seanlip
-/core/domain/html*.py @seanlip
-/core/domain/rights_manager*.py @seanlip
-/core/domain/role_services*.py @seanlip
-/core/storage/ @seanlip
-/core/domain/user*.py @seanlip
-/requirements.txt @vojtechjelinek
-/requirements.in @vojtechjelinek
-/requirements_dev.txt @vojtechjelinek
-/requirements_dev.in @vojtechjelinek
-/dependencies.json @vojtechjelinek
-/package.json @vojtechjelinek
-/yarn.lock @vojtechjelinek
+/core/templates/services/svg-sanitizer.service.ts @oppia/core-reviewers
+/scripts/linters/warranted_angular_security_bypasses.py @oppia/core-reviewers
+/core/controllers/access_validators*.py @oppia/core-reviewers
+/core/controllers/acl_decorators*.py @oppia/core-reviewers
+/core/controllers/base*.py @oppia/core-reviewers
+/core/domain/html*.py @oppia/core-reviewers
+/core/domain/rights_manager*.py @oppia/core-reviewers
+/core/domain/role_services*.py @oppia/core-reviewers
+/core/storage/ @oppia/core-reviewers
+/core/domain/user*.py @oppia/core-reviewers
+/requirements.txt @oppia/data-and-stability-reviewers
```

Comment:
Would it be possible to reorder lines so that anything owned by @oppia/core-reviewers and @oppia/web-android-compatibility-reviewers goes at the bottom of the whole file, to avoid those lines getting overwritten by other rules in the future?

Also, maybe we should have a lint check or backend test that ensures that every line's assignees starts with "@oppia/" to prevent individual assignment in the future -- what do you think?

Reply:
done